public/javascripts/SVValidate/src/keyboard/Keyboard.js

```diff
@@ -5,7 +5,9 @@ function Keyboard(menuUI) {
         disableKeyboard: false,
         keyPressed: false,
         shiftDown: false,
-        addingComment: false
+        addingComment: false,
+        disagreeSelected: false,
```

Comment:
You don't need to add new variables to track whether disagree/unsure are selected. This info is being tracked elsewhere, which you can find if you look at what happens in the yes/no/unsure buttons' click funcs.

Reply:
done